#heartbeat
time stamp
string HostName
string BaseNodeName
string NodeName
uint8 NodeState
